nrfx_config.h

```diff
@@ -48,6 +48,22 @@
 #define NRFX_CLOCK_ENABLED 1
 #endif
 
+#ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC
+#define NRFX_CLOCK_CONFIG_LF_SRC 0
+#elif CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL
+#define NRFX_CLOCK_CONFIG_LF_SRC 1
+#elif CONFIG_CLOCK_CONTROL_NRF_K32SRC_SYNTH
+#define NRFX_CLOCK_CONFIG_LF_SRC 2
```

Comment:
For nRF5340, these values should be 1, 2, and 3, respectively. For nRF9160 similarly, except for the Synth option not available.

Reply:
> For nRF5340, these values should be 1, 2, and 3, respectively. Similarly for nRF9160, except for the Synth option not available.

Reply:
done